packages/squiggle-lang/src/fr/tag.ts

```diff
@@ -54,6 +54,27 @@ function _ensureTypeUsingLambda<T1>(
   }
 }
 
+const booleanTagDefs = (tagName: string) => [
```

Comment:
Here's a helper for you:

```typescript
type PickByValue<T, ValueType> = NonNullable<keyof Pick<T, {
    [Key in keyof T]: T[Key] extends (ValueType | undefined) ? Key : never;
}[keyof T]>>;
```

Then you can use `tagName: PickByValue<ValueTagsType, boolean>`.

Reply:
Thanks! I feel like this might be a bit overly-clever for the value here, but happy to add now.